infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/config/LogReplicationConfig.java

```diff
@@ -60,16 +61,18 @@ public abstract class LogReplicationConfig {
     // Snapshot Sync Batch Size(number of messages)
     private int maxNumMsgPerBatch;
 
-    // Max Size of Log Replication Data Message
-    private int maxMsgSize;
-
     // Max Cache number of entries
     private int maxCacheSize;
 
     /**
-     * The max size of data payload for the log replication message.
+     * The max size of replicated data payload transferred at a time.
+     */
+    private int maxTransferSize;
```

Comment:
can we please make this a long to avoid any computational overflows accidentally?